### PR TITLE
Refactor TemporalGraph and snapshotAtTime for Landy

### PR DESCRIPTION
**Pull request by TheodorRene**
The snapshotAtTime for Landy is currently failing

### CODE REVIEW

File: data-generation/src/main/scala/thesis/Landy.scala [hunk 1]
```diff
@@ -12,28 +12,44 @@ import scala.collection.mutable
...
     Graph(vertices, edges)
   }
 
+  override def snapshotAtTime(instant: Instant): Snapshot = {
+    val vertices = localVertices
+      .filter(vertex =>
+        vertex._2.validFrom < instant &&
+          vertex._2.validTo >= instant
+      )
+      .mapValues(_.attributes)
```
Comment (eivindkopperud) [inline data-generation/src/main/scala/thesis/Landy.scala]:
```suggestion
      .map(vertex => (vertex._2.id, vertex._2))
      .mapValues(_.attributes)
```
Vil dette funke? Fordi med .mapValues vil du vel fremdeles beholde de lokale nodene sine id-er?

**Comment by TheodorRene** [comment]
wtf,

**Comment by eivindkopperud** [comment]
> wtf,

0 commits nice

**Comment by TheodorRene** [comment]
Close for now as it is bugged and scuffed

**Pull request closed without merging**